packages/mcp-docs-server/src/tools/docs.ts

```diff
@@ -179,11 +198,12 @@ export const docsTool = {
                 error: null,
               };
             }
-            const suggestions = await findNearestDirectory(path, availablePaths);
+            const directorySuggestions = await findNearestDirectory(path, availablePaths);
+            const contentBasedSuggestions = await getMatchingPaths(path, args.queryKeywords ?? [], docsBaseDir);
             return {
               path,
               content: null,
-              error: suggestions,
+              error: [directorySuggestions, contentBasedSuggestions].join('\n\n'),
```

Comment:
do we need to do something like:
```suggestion
              error: [...directorySuggestions, ...contentBasedSuggestions].join('\n\n'),
```